src/function/handler.ts

```diff
@@ -7,6 +7,6 @@ export interface HandlerCallback {
   (error: any, response: Response): void
 }
 
-export interface Handler {
-  (event: Event, context: Context, callback: HandlerCallback): void | Response | Promise<Response>
+export interface Handler<C extends Context> {
```

Comment:
```suggestion
export interface Handler<C extends Context = Context> {
```

This needs a default